### PR TITLE
A form page

### LINKED ISSUE #2: A form page
A form page from where users can submit a source or resource easily.
The form should include:-
- Name input text field for the user [**mandatory**]
- User email contact input text field [**optional**]
- Dropdown to select source or resource [**mandatory**]
- Name input text field for source or resource [**mandatory**]
- Source or resource URL input text field [**mandatory**]
- Description text area for source or resource [**mandatory**]

### PR DESCRIPTION
**Pull request by MuhammadJufri**
Closes #2 

### What's being changed:

1. Responsive
2. Add form page
3. Add JSON Server
4. Update JSON file data & Update package.json file
5. Shorten code

### CODE REVIEW

**Comment by MuhammadJufri** [comment]
And also you have to run the JSON server first before you add a new resource

To run the JSON server you can just simply write:

``` npm run startServer ```

File: package.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
```
Comment (MuhammadJufri) [inline package.json:9]: Install Axios

Comment (sarvesh4396) [reply package.json:9]: Thank you @MuhammadJufri, Can you please convert all strings to double quotes?
If there is a reason to do that please let me know.

Comment (MuhammadJufri) [reply package.json:9]: I've changed them to double quotes.
I'm not sure what you mean if there is a reason to do that

Comment (sarvesh4396) [reply package.json:9]: Thank you.
I meant if any standard practice you are following.
I will review it today.

Comment (MuhammadJufri) [reply package.json:9]: Well I preferred to use double quotes because single quotes sometimes can look like a backtick

**Pull request merged**